tasks/test.py

```diff
@@ -678,7 +678,7 @@ def lint_python(ctx):
 
     ctx.run("flake8 .")
     ctx.run("black --check --diff .")
-    ctx.run("isort --check-only --diff .")
+    ctx.run("isort --check-only --skip procfs --diff .")
```

Comment:
Why do we need this?

Reply:
TLDR I will remove this as it's not more relevant

We would need this because (running on circle ci as root) isort follow the link and scan the full repo and been hook by our procfs test directory that contain symlink like procfs/1/root -> /  isort continue his scanning and try to read procfs/1/root/dev/console and get stuck there on read()

* We running as root in circleci as some tests needed
* I think isort should be patched to not open file that are not regular or not symlink (it should avoid character/block/pipe/unix socket devices

We don't need the patch to make the CI happy as I reduce the testing procfs files used to the minimum for our tests

Reply:
I see, thanks for explaining!